datatransfer/tasks.py

```diff
@@ -177,10 +209,16 @@ def process_files(source=settings.INGEST_SOURCE_PATH,
         try:
             contents = read_storage.read_file(file_name)
             write_storage.write_file(file_name, contents)
+            if settings.WRITE_MQ.capitalize() == "True" and not settings.WRITE_STORAGE_TYPE.endswith(('SftpStorage', 'FolderStorage')):
+                mq.publish_event(file_name)
```

Comment:
Is this providing an 'all or nothing' approach, I couldn't see any handling for the ACK from the Queue?